pkg/config/config.go

```diff
@@ -319,9 +319,12 @@ type Config struct {
 		FixedID string `mapstructure:"fixed_id"`
 	} `mapstructure:"installation"`
 	Security struct {
-		AuditCheckInterval time.Duration `mapstructure:"audit_check_interval"`
-		AuditCheckURL      string        `mapstructure:"audit_check_url"`
+		CheckLatestVersion      bool          `mapstructure:"check_latest_version"`
+		CheckLatestVersionCache time.Duration `mapstructure:"check_latest_version_cache"`
```

Comment:
I think it might make more sense to name `CheckLatestVersionCache` in the same way as the audit check, i.e. `CheckLatestVersionInterval`

Reply:
Why name it the same? It's not working the same, one is an active periodic interval, the other is a cache that can stay stale forever if none calls it.